modules/manifold/middleware.js

```diff
@@ -1 +1 @@
-module.exports = require('./dist/esm/middleware');
+module.exports = require('./dist/middleware');
```

Comment:
If there's only one build, it should be possible to remove these proxy files entirely. Please evaluate the possibility (trying building, publishing and installing if necessary)